packages.md

```diff
@@ -117,6 +117,22 @@ above) will **not** be available any more after renaming. For example:
 This might cause problems if you want to load another package that relies on
 the modified package.
 
+### Package Local Nicknames (PLN)
+Many implementations (SBCL, CCL, ECL, Clasp, ABCL, ACL, LispWorks >= 7.2) support Package Local Nicknames (PLN). 
```

Comment:
CLISP also supports them if built with the yet-unmerged commits from https://gitlab.com/gnu-clisp/clisp/-/merge_requests/3